initializr-service/src/main/resources/application.yml

```diff
@@ -91,6 +96,96 @@ initializr:
+          groupId: com.microsoft.azure
+          artifactId: azure-mediaservices-spring-boot-starter
+          description: Spring Boot starter for Azure Media Services
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot-starters
+              description: Source code and README for the starter
+        - name: Azure Service Bus
+          id: azure-service-bus
+          groupId: com.microsoft.azure
+          artifactId: azure-servicebus-spring-boot-starter
+          description: Spring Boot starter for Azure Service Bus
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot-starters
+              description: Source code and README for the starter
+        - name: Azure Document DB
+          id: document-db
+          groupId: com.microsoft.azure
+          artifactId: azure-documentdb-spring-boot-starter
+          description: Spring Boot starter for Azure Document DB service
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot-starters
+              description: Source code and README for the starter
+        - name: Azure Active Directory
+          id: active-directory
+          groupId: com.microsoft.azure
+          artifactId: azure-ad-integration-spring-boot-starter
+          description: Spring Security integration with Azure Active Directory for authentication
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot-starters
+              description: Source code and README for the starter
+        - name: Microsoft Graph
+          id: microsoft-graph
+          groupId: com.microsoft.azure
+          artifactId: microsoft-graph-spring-social-starter
+          description: Spring Social integration with Microsoft Graph
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot-starters
+              description: Source code and README for the starter
+        - name: Azure Key Vault
+          id: key-vault
+          groupId: com.microsoft.azure
+          artifactId: azure-keyvault-secrets-spring-boot-starter
+          description: Spring value annotation integration with Azure Key Vault Secrets
+          versionRange: 1.5.4.RELEASE
+          bom: azure
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot-starters
+              description: Source code and README for the starter
+        - name: Azure Cloud Foundry Service
```

Comment:
This entry does not provide auto-configuration so this feature should be refactored somehow. See also https://github.com/Microsoft/azure-spring-boot-starters/issues/158